makefile

```diff
@@ -121,3 +121,17 @@ validate-a11y: install # PRIVATE
 # Take screenshots for a visual check.
 screenshots: install
 	@./tools/task-runner/runner screenshot
+
+# *********************** COMMERCIAL ***********************
+# Do all the commercial things
+commercial: install
+	@./tools/task-runner/runner commercial --verbose
```

Comment:
i'm sorry you had to write code in here. possibly the worst thing i've done here! ☠️

Reply:
I didn’t really have to, but I needed to understand how `task-runner` did its thing, so it was as good a place as any.